src/components/templates/HackathonSingle.js

```diff
@@ -203,14 +205,14 @@ const HackathonSingle = ({ initialState = defaultState }) => {
     endDateArr[2] + "-" + endDateArr[1] + "-" + endDateArr[0];
 
   // Event is open or closed for registration
-  let dateNow = Date.now(); 
+  let dateNow = Date.now();
   let startDateInMs = new Date(startDate).getTime();
   let daysToEvent = Math.floor((startDateInMs - dateNow) / (1000 * 3600 * 24));
   window.localStorage.setItem("closingDate", JSON.stringify(eventIsOpen));
```

Comment:
Why are you saving this in localStorage?

Reply:
That's mildred's part. I didn't touch it.
Only wired everything with conditional statements